README.md

```diff
@@ -1,25 +1,25 @@
+**To see how the Neural Network performs after a sufficiently timed training session, see the test accuracies listed in the [TENSORFLOW-Cognitive-State-Detection](https://github.com/stevebottos/TENSORFLOW-Cognitive-State-Detection) repository.**
 
 # Introduction
-My graduate advisor worked in the past with a prominent U.S. entity (which shall remain nameless) which was interested in monitoring the cognitive context of its employees through non-invasive means in order to better delegate tasks and reduce human error during human/machine interaction. Previous studies have related pupil diameter to the level of focus which an individual is currently experiencing. The entity had developed a test "game" which contained three categories of tasks: easy, somewhat difficult, or very difficult. The goal was to collect pupil diameter data recorded throughout the test from various individuals and accurately predict when the individual was very focused vs. not very focused - assuming that the majority of people would be very focused during each very difficult task and vice-versa for the easy tasks. In practice however, the mean and standard deviation of each participant's pupil diameters during each task hardly show any differences during each task. Results were somewhat lackluster and little was done to push the project forward. I thought that I would be able to improve on their results by measuring alternative characteristics of the data and performing a few feature enginering steps before finally feeding input data through a Feed Forward Neural Network.<br/>
+My graduate advisor worked in the past with a prominent U.S. entity (which shall remain nameless) which was interested in monitoring the cognitive context of its employees through non-invasive means in order to better delegate tasks and reduce human error during human/machine interaction. Previous studies have related pupil diameter to the level of focus which an individual is currently experiencing. The entity had developed a test "game" which contained three categories of tasks: easy, somewhat difficult, or very difficult. The goal was to collect pupil diameter data recorded throughout the test from various individuals and accurately predict when the individual was very focused vs. not very focused - assuming that the majority of people would be very focused during each very difficult task and vice-versa for the easy tasks. In practice, however, the mean and standard deviation of each participant's pupil diameters during each task hardly show any differences during each task. Results were somewhat lackluster and little was done to push the project forward. I thought that I would be able to improve on their results by measuring alternative characteristics of the data and performing a few feature engineering steps before finally feeding input data through a Feed Forward Neural Network.

```

Comment:
Add comma: "In practice however," -> "In practice, however,". And remove superfluous `<br/>` from end of paragraph.